cob_navigation_global/CMakeLists.txt

```diff
@@ -16,12 +16,19 @@ install(DIRECTORY launch config
 #############
 ## Testing ##
 #############
-
+SET(robotlist
+    cob4-2
+    cob4-5
+    cob4-7
+    raw3-1
+    raw3-3
+    raw3-5
+)
+    
 if(CATKIN_ENABLE_TESTING)
-  find_package(cob_supported_robots REQUIRED)
   find_package(cob_default_env_config REQUIRED)
   find_package(roslaunch REQUIRED)
-  foreach(robot ${cob_supported_robots_ROBOTLIST})
```

Comment:
because `cob_supported_robots` supports `cob4-3`, `cob4-4` and `cob4-6` which we do not support nav-configs for...